doc/classes/ParallaxBackground.xml

```diff
@@ -108,16 +108,22 @@
 	</methods>
 	<members>
 		<member name="scroll_base_offset" type="Vector2" setter="set_scroll_base_offset" getter="get_scroll_base_offset">
+			Base position offset of all [ParallaxLayer] childs.
 		</member>
 		<member name="scroll_base_scale" type="Vector2" setter="set_scroll_base_scale" getter="get_scroll_base_scale">
+			Base motion scale of all [ParallaxLayer] childs.
 		</member>
 		<member name="scroll_ignore_camera_zoom" type="bool" setter="set_ignore_camera_zoom" getter="is_ignore_camera_zoom">
+			If [code]true[/code] elements in [ParallaxLayer] child aren't affected by the zoom level of the camera.
 		</member>
 		<member name="scroll_limit_begin" type="Vector2" setter="set_limit_begin" getter="get_limit_begin">
+			Top left limits for scrolling to begin. If the camera is outside of this limit the background will not continue to scroll. Must be lower than [member scroll_limit_end] to work.
 		</member>
 		<member name="scroll_limit_end" type="Vector2" setter="set_limit_end" getter="get_limit_end">
+			Right bottom limits for scrolling to end. If the camera is outside of this limit the background will not continue to scroll. Must be higher than [member scroll_limit_begin] to work.
 		</member>
 		<member name="scroll_offset" type="Vector2" setter="set_scroll_offset" getter="get_scroll_offset">
+			Scroll value of the [code]ParallaxBackground[/code]. Can be used to manually manage scroll. This value is overridden if a [Camera2D] is used.
```

Comment:
- no [code] tags around class name.

The ParallaxBackground's scroll value. Calculated automatically when using a [Camera2D], but can be used to manually manage scrolling when no camera is present.